src/views/IncorporationAgreement.vue

```diff
@@ -0,0 +1,316 @@
+<template>
+  <div>
+    <section class="mt-10">
+      <header id="share-structure-header">
+        <h2>1. Incorporation Agreement and Articles</h2>
+      </header>
+      <p>
+        Before submitting your incorporation application you <b>must complete, sign, and date</b> an
+        <v-tooltip top color="primary">
+          <template v-slot:activator="{ on }">
+            <span v-on="on" class="tool-tip"> Incorporation Agreement</span>
+          </template>
+          <span>
+            A document that is signed and dated by the people who agree to form the corporation. They are the
+            incorporators and the first shareholders of the corporation.
+          </span>
+        </v-tooltip>
+        , and a set of
+        <v-tooltip top color="primary">
+          <template v-slot:activator="{ on }">
+            <span v-on="on" class="tool-tip"> Benefit Company Articles</span>
+          </template>
+          <span>
+            The Articles for a benefit company must state the benefits the company intends to provide to society, as
+            well as outlining the rules and procedures for corporate matters such as holding meetings, issuing and
+            transferring shares, and duties of directors and officers.
+          </span>
+        </v-tooltip>
+        containing a
+        <v-tooltip top color="primary">
+          <template v-slot:activator="{ on }">
+            <span v-on="on" class="tool-tip"> benefit provision</span>
```

Comment:
Benefit Provision ?

Reply:
Not sure. Will be checking with Linda/Scott

Reply:
Checked with Linda and Scott. It is benefit provision